portal/patient_self_checkin/mySQL_Connector.php

```diff
@@ -0,0 +1,13 @@
+<?php
+$ignoreAuth = true;
+require_once "../../interface/globals.php";
+require_once $GLOBALS['webserver_root'] . '/sites/default/sqlconf.php';
```

Comment:
Including `interface/globals.php` will give you access to `library/sql.inc` which is our central point of access to the database. https://github.com/openemr/openemr/blob/master/library/sql.inc